index.html

```diff
@@ -1,57 +1,69 @@
 <!doctype html>
+<style>
```

Comment:
since we're using css, you don't actually need to do the style here! what you have in your css should be sufficient.